libs/utils/src/lib/helpers/functions.ts

```diff
@@ -937,3 +937,12 @@ export function findAncestorElement(
 export function hasInvalidChars(str: string, invalidChars: string[]): boolean {
   return invalidChars.some((invalidChar) => includes(str, invalidChar));
 }
+
+/**
+ * Returns true if true or "true"
+ * @arg {any} value - A value to compare.
+ * @return {boolean} - True if true or "true"
+ */
+export function isTrue(value: any): boolean {
+  return value === true || value === 'true';
```

Comment:
you should check based on typeof then... to keep this method generic, because i could say 0 or 1 also to be boolean when it is integer